tracing-core/src/lib.rs

```diff
@@ -114,7 +114,7 @@
     unused_parens,
     while_true
 )]
-#[cfg(not(feature = "std"))]
+
 extern crate alloc;
```

Comment:
do you know if we still need an `extern crate` declaration to reference liballoc? if we don't, we can probably just remove this.

Reply:
Yes, we do. Here's the error without:

```
error[E0433]: failed to resolve: use of undeclared type or module `alloc`
 --> tracing-core/src/callsite.rs:9:5
  |
9 | use alloc::vec::Vec;
  |     ^^^^^ use of undeclared type or module `alloc`
```